docs/json.md

```diff
@@ -84,6 +84,19 @@ HEMTT will copy the files to the release directory after a successful release bu
 ```
 <hr/>
 
+## include
+**Type**: Array \[String (Path)\]
+
+HEMTT will include matching relative or absolute paths when building.
+
+```json
+"include": [
+    "./include"
+]
+```
+
+`./include` will be automatically added on project creation if "include" folder is present.
```

Comment:
We can remove that functionality if wanted though, or change that one to be checked in runtime, but I don't like that.

Reply:
Maybe use `-include` inside the hemtt.json file to specify not including it?

Reply:
It will only be added if the actual folder is present during `hemtt create`, and actually added to JSON itself. So if projects removes it they just remove it from the JSON.

Reply:
Ah ok, didn't notice on project creation